specification/maps/resource-manager/Microsoft.Maps/preview/2023-12-01-preview/examples/PrivateLinkResources_List.json

```diff
@@ -0,0 +1,32 @@
+{
+  "parameters": {
+    "api-version": "2023-12-01-preview",
+    "subscriptionId": "21a9967a-e8a9-4656-a70b-96ff1c4d05a0",
+    "resourceGroupName": "myResourceGroup",
+    "accountName": "myMapsAccount",
+    "privateEndpointConnectionName": "{privateEndpointConnectionName}",
+    "monitor": "true"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "id": "/subscriptions/21a9967a-e8a9-4656-a70b-96ff1c4d05a0/resourceGroups/myResourceGroup/providers/Microsoft.Maps/accounts/myMapsAccount/privateLinkResources/mapsAccount",
+            "name": "mapsAccount",
+            "type": "Microsoft.Maps/accounts/privateLinkResources",
+            "properties": {
+              "groupId": "mapsAccount",
+              "requiredMembers": [
+                "myMapsAccount"
+              ],
+              "requiredZoneNames": [
+                "myMapsAccount.privatelink.{accountLocation}.maps.azure.com"
+              ]
```

Comment:
We should make sure this is what we expect in [our design](https://msazure.visualstudio.com/DefaultCollection/One/_git/Azure-IoT-LocationBasedServices?path=/src/ResourceProvider/ResourceProviderBusinessLayer/PrivateEndpoints/PrivateLinkResources/PrivateLinkResourceResultBuilder.cs&version=GBmaster).